training/distributed_training/pytorch/model_parallel/gpt-j/train_gptj_smp_notebook.ipynb

```diff
@@ -0,0 +1,438 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #14.    print(f"SageMaker Execution Role:{role}")
</pre><p>Nit: add a space after all colons in the print statements in this cell (3 occurrences)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3158/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>